trace/tracestate.go

```diff
@@ -260,6 +260,16 @@ func (ts TraceState) Get(key string) string {
 	return ""
 }
 
+// Walk walks all key value pairs in the TraceState by calling f
+// Iteration stops if f returns false.
+func (ts TraceState) Walk(f func(key, value string) bool) {
+	for _, m := range ts.list {
+		if !f(m.Key, m.Value) {
+			break
```

Comment:
This `break` is never tested.
See https://github.com/open-telemetry/opentelemetry-go/blob/2726c6aab72bd503bc0ace4b63c7202853902229/log/record_test.go#L84-L93 for inspiration.

Reply:
Updated.